src/me.rs

```diff
@@ -14,12 +14,77 @@ use partition::*;
 use plane::*;
 use FrameInvariants;
 use FrameState;
+use util::*;
+
+use libc;
+
+#[cfg(all(any(target_arch = "x86", target_arch = "x86_64"), not(windows)))]
+extern {
+  fn rav1e_sad_4x4_ssse3(
+    src: *const u16, src_stride: libc::ptrdiff_t, dst: *const u16,
+    dst_stride: libc::ptrdiff_t
+  ) -> u32;
+
+  fn rav1e_sad_8x8_ssse3(
+    src: *const u16, src_stride: libc::ptrdiff_t, dst: *const u16,
+    dst_stride: libc::ptrdiff_t
+  ) -> u32;
+
+  fn rav1e_sad_16x16_ssse3(
+    src: *const u16, src_stride: libc::ptrdiff_t, dst: *const u16,
+    dst_stride: libc::ptrdiff_t
+  ) -> u32;
+
+  fn rav1e_sad_32x32_ssse3(
+    src: *const u16, src_stride: libc::ptrdiff_t, dst: *const u16,
+    dst_stride: libc::ptrdiff_t
+  ) -> u32;
+}
+
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
```suggestion
#[cfg(all(any(target_arch = "x86", target_arch = "x86_64"), not(windows)))]
```

You forgot about windows :)